src/cljs/main/broadfcui/main.cljs

```diff
@@ -120,68 +120,83 @@
      (utils/ajax {:url (config/alerts-json-url)
                   :headers {"Cache-Control" "no-store, no-cache"}
                   :on-done (fn [{:keys [status-code raw-response]}]
                              (if (utils/check-server-down status-code)
-                               (if (>= (:failedRetries @state) (config/max-retry-attempts))
-                                 (swap! state assoc :alerts [{:title "Google Service Alert"
-                                                            :message "There may be problems accessing data in Google Cloud Storage."
-                                                            :link "https://status.cloud.google.com/"}])
-                                 (swap! state assoc :failedRetries (+ (:failedRetries @state) 1)))
+                               (if (>= (:failed-retries @state) (config/max-retry-attempts))
+                                 (swap! state assoc :service-alerts
+                                        [{:title "Google Service Alert"
+                                          :message "There may be problems accessing data in Google Cloud Storage."
+                                          :link "https://status.cloud.google.com/"}])
+                                 (swap! state assoc :failed-retries (+ (:failed-retries @state) 1)))
                                (let [[parsed _] (utils/parse-json-string raw-response true false)]
+                                 (utils/parse-json-string raw-response true false)
                                  (if (not-empty parsed)
-                                   (swap! state assoc :alerts parsed  :failedRetries 0)
-                                   (swap! state dissoc :alerts)))))}))})
+                                   (swap! state assoc :service-alerts parsed :failed-retries 0)
+                                   (swap! state dissoc :service-alerts)))))}))})
+
+(defn- show-js-exception [e]
+  (comps/push-ok-cancel-modal
+   {:header [:span {} (icons/icon {:style {:color (:warning-state style/colors)
+                                           :marginRight "1rem"}}
+                                  :warning)
+             "Something Went Wrong"]
+    :content [:div {:style {:width 800}}
+              "There was an error in FireCloud. It may not mean anything, but you should consider reloading the page to be safe."
+              [:div {:style {:marginTop "1rem"}} "Here are some details about the error that occurred."
+               " If you post this on our "
+               [:a {:href (config/forum-url)
+                    :target "_blank" :style {}} "forum"] ", our team can take a look."]
+              [:div {:style {:fontFamily "monospace" :whiteSpace "pre" :overflow "auto"
+                             :backgroundColor "black" :color "white"
+                             :padding "0.5rem" :marginTop "0.5rem" :borderRadius "0.3rem"}}
+               [:div {:style {:fontWeight "bold"}} "Error: "]
+               (.-message e)
```

Comment:
I'd prefer we use `(aget e "message")` for JavaScript object accessors because they will never get munged by Closure's advanced compilation (even if we are not able to use a externs file).